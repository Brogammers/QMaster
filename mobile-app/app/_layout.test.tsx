import React from 'react';
import renderer from 'react-test-renderer';

import RootLayout from './_layout'; // Import the correct component

describe('<RootLayout />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<RootLayout />).toJSON();
    expect(tree).toMatchSnapshot(); // Use snapshot testing for UI consistency
  });

  it('renders RootLayoutNav when fonts are loaded', () => {
    const tree = renderer.create(<RootLayout />).toJSON();
    expect(tree).toMatchSnapshot(); // Snapshot should include RootLayoutNav
  });

  // Add more tests as needed, e.g., for loading states, error handling, etc.  
});
