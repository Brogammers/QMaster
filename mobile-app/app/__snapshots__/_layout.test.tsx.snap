// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootLayout /> renders RootLayoutNav when fonts are loaded 1`] = `null`;

exports[`<RootLayout /> renders correctly 1`] = `null`;
